core/sqf/sql/scripts/get_libhdfs_files

```diff
@@ -110,113 +123,141 @@ if [[ $FORCE_BUILD == true || \
       exit 1
     fi
   fi
-
   cd $LIBHDFS_TEMP_DIR
 
-  PROTOBUF_VER=`protoc --version 2>/dev/null | cut -f 2 -d ' '`
+  if [[ $SOURCE_BUILD != true ]]; then
+    echo "Downloading Hadoop-common binary distro..." | tee -a ${LOGFILE}
```

Comment:
There might be a small chance that we would pick up 32 bit libraries, since older Hadoop binary packages contain those instead of 64 bit libraries. Probably ok, as we move away from those older versions.